ui/components/component-library/popover/popover.tsx

```diff
@@ -39,6 +39,7 @@ export const Popover: PopoverComponent = React.forwardRef(
       title,
       isPortal = false,
       arrowProps,
+      onClickOutside,
```

Comment:
Non blocking: Curious as to why you chose to name the prop `onClickOutside` rather than `onClose` in the PR description you've just stated the functionality to "give the dev more flexibility on the state of when the Popover should be opened and closed." but can you foresee other functionality outside of closing the Popover?

Reply:
But aren't you passing the function to close the the Popover in the story? It's similar to how it works in the modal component https://github.com/MetaMask/metamask-extension/blob/develop/ui/components/component-library/modal/modal.stories.tsx#L119